pkgs/top-level/python-packages.nix

```diff
@@ -4932,6 +4950,8 @@ in {
     else
       callPackage ../development/python-modules/mutagen { };
 
+  mutatorMath = callPackage ../development/python-modules/mutatorMath { };
```

Comment:
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

Reply:
What about the `name` attribute? Should it be like the attribute name or like in the pypi archive (so `inherit pname` is possible)?

Reply:
pname is fine to keep with the irregular format, so that it agrees with the fetchpypi usage

Reply:
the main  thing is that referencing the package should be done through the normalized name (e.g. `propagatedBuildInputs = [ mutatormath ];`

Reply:
Then, all should be well now.